dev-packages/e2e-tests/test-applications/node-express-esm-preload/tests/server.test.ts

```diff
@@ -117,9 +117,7 @@ test('Should record a transaction for route with parameters', async ({ request }
   });
 });
 
-// This fails https://github.com/getsentry/sentry-javascript/pull/12587#issuecomment-2181019422
-// Skipping this for now so we don't block releases
-test.skip('Should record spans from http instrumentation', async ({ request }) => {
+test('Should record spans from http instrumentation', async ({ request }) => {
```

Comment:
not sure if this is fixed by this specifically or something else, but this seems to work again now.